### PR TITLE
Use message service in language client

### PR DESCRIPTION
**Pull request by AlexTugarev**
See #185

Signed-off-by: Alex Tugarev <alex.tugarev@typefox.io>

### CODE REVIEW

**Comment by AlexTugarev** [comment]
@akosyakov, I've tested this with java extension. It tries to execute the selected command, which is unfortunately not registered.

**Comment by akosyakov** [comment]
@AlexTugarev it is cool, the issue with the command is unrelated

File: packages/languages/src/common/console-window.ts [hunk 1]
```diff
@@ -5,25 +5,35 @@
...
+        const originalActions = new Map((actions || []).map(action => [action.title, action] as [string, T]));
+        const actionTitles = (actions || []).map(action => action.title);
+        const mapActionType: (result: string | undefined) => (T | undefined) = result => {
+            if (!!result) {
+                return originalActions.get(result);
+            }
+            return undefined;
+        };
         if (type === MessageType.Error) {
-            console.error(message);
```
Comment (akosyakov) [inline packages/languages/src/common/console-window.ts]: I think we should rename `ConsoleWindow` to something else, like `WindowImpl`

Comment (akosyakov) [reply packages/languages/src/common/console-window.ts]: and the file as well

Comment (AlexTugarev) [reply packages/languages/src/common/console-window.ts]: done

**Pull request merged**